docs/events/com.mbta.rtr.lightrail-vehicle-information.mdx

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+import LightRailVehicleInformation from "@site/examples/com.mbta.rtr.light-vehicle-information/LightRailVehicleInformation.json";
```

Comment:
I tried to run this locally. For some reason I'm getting this error:

> Module not found: Error: Can't resolve '@site/examples/com.mbta.rtr.light-vehicle-information/LightRailVehicleInformation.json' in '/{...local path...}/docs/events'

But I can't figure out why. If it works on your machine, then don't worry about it.

Reply:
Oh, there is a typo, and it's not [getting tested in CI](https://github.com/mbta/schemas/pull/12/checks#step:4:40).
> skipping examples/com.mbta.rtr.lightrail-vehicle-information/, no matching JSON Schema in schemas/com.mbta.rtr.lightrail-vehicle-information.json

This line says `light-vehicle`. The examples directory is `lightrail-vehicle`. And the event name is `light-rail-vehicle`.

Reply:
ahhh, good catch! I was just running `yarn ci` locally not the actual application. `yarn start` does not work for me.